plugins/azure/containerapps/containerAppManagedIdentity.js

```diff
@@ -0,0 +1,56 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Containe Apps Managed Identity',
+    category: 'Container Apps',
+    domain: 'Containers',
+    severity: 'Medium',
+    description: 'Ensure that Azure Container Apps has managed identity enabled.',
+    more_info: 'Enabling managed identity for Azure Container Apps automates credential management, enhancing security by avoiding hard-coded credentials and simplifying access control to Azure services.',
+    recommended_action: 'Enable system or user-assigned identities for all Azure Container Apps.',
```

Comment:
if we can also add managed identity after creating resource, recommended action should be Modify Container apps and add managed identity.